pages/api-reference/accounts/create-a-funding-source.mdx

```diff
@@ -11,19 +11,32 @@ import guideIcon from "../../../assets/images/content-images/content-icons/guide
 
 # Create a funding source for an account
 
-This section details how to add a bank account to a Dwolla Master Account. The bank account will have a status of `unverified` upon creation. Before a Dwolla Master Account is eligible to transfer money from their bank or credit union account they need to verify ownership of the account via micro-deposit verification.
+This section details how to add a bank account and a `VAN` to a Dwolla Master Account.
```

Comment:
Let's spell out Virtual Account Numbers in this line since it seems to be the first appearance of "VAN" in API reference.